### PR TITLE
Make displaying script type a little more clear

### PR DESCRIPTION
**Pull request by benthecarman**
Changed order to input, change, output as well as made the names of script types better than just what they are named in bitcoin-s

### CODE REVIEW

File: components/Status.tsx [hunk 1]
```diff
@@ -81,3 +81,23 @@ export default function Status({ statusData, coordinatorName }) {
...
+    case 'witness_v0_scripthash':
+      return 'P2WSH';
+    case 'witness_v1_taproot':
+      return 'Taproot';
+    case 'scripthash':
+      return 'P2SH';
+    case 'pubkeyhash':
+      return 'P2PKH';
+    case 'pubkey':
+      return 'P2PK';
```
Comment (benthecarman) [inline components/Status.tsx]: Do you think it'd be good to put an example prefix for the address too?

Ie "P2WPKH (bc1q...)"

Comment (deregs) [reply components/Status.tsx]: Yeah I think that would be a helpful addition

File: components/Status.tsx [hunk 2]
```diff
@@ -81,3 +81,23 @@ export default function Status({ statusData, coordinatorName }) {
     </li>
   );
 }
+
+
+function scriptTypeToString(scriptType: string): string {
```
Comment (deregs) [inline components/Status.tsx]: this function can go in `utils/convertor.ts` and then we can also use it in the `InputType` component

Comment (benthecarman) [reply components/Status.tsx]: Done

File: components/Status.tsx (hunk 1 above)
Comment (benthecarman) [reply components/Status.tsx]: Added. scary that copilot autofilled some of these for me

**Review by deregs — APPROVED** [review]

**Pull request merged**